sys/arm64/include/cherireg.h

```diff
@@ -91,8 +91,13 @@
 	(CHERI_PERM_GLOBAL | CHERI_PERM_LOAD | CHERI_PERM_LOAD_CAP |	\
 	(CHERI_PERMS_SWALL & ~CHERI_PERM_CHERIABI_VMMAP))
 
+/*
+ * XXX Will probaby want non-executive PCC when we add comparmentalization
+ * support, but make all code caps executive for now.
+ */
```

Comment:
I think this will be needed even then since all code mappings need it (at least initially)? I guess userspace will have to be careful to clear it and not allow mmap() directly in compartments?

Reply:
In the libcheri world yes, the non-sandboxed part of the address space gets ASR/Executive, and libcheri clears it for compartments. I don’t know how things are done in the coprocess world though.